blues/simulation.py

```diff
@@ -323,7 +342,7 @@ def _getSimulationInfo(self):
         self.log.info('Total Simulation Time = %s ps' % totaltime)
         self.log.info('\tTotal NCMC time = %s ps' % (int(timeNC) * int(self.opt['nIter'])))
         self.log.info('\tTotal MD time = %s ps' % (int(timeMD) * int(self.opt['nIter'])))
-
+        self.totalNC = totalNC
```

Comment:
What is this doing? Not obvious to me why this addition is coming in here, just wanted to make sure it's not a typo.

Reply:
I was using this for printing out the progress of the NCMC part, but ultimately left out the progress printing. I could delete this line.